content/documentation/terminology/index.adoc

```diff
@@ -0,0 +1,167 @@
+---
+title: "Terminology"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "documentation"
+---
+
+= Kiali Overview
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Terminology Content
+:keywords: Kiali Documentation
+:icons: font
+:imagesdir: /images/documentation/terminology/
+
+== Application
+
+It's a runtime/program that runs in the environment.
+
+== App Name
+
+It's the name of the <<Application>> deployed in your environment.
```

Comment:
Also, I would add that the "App Name" comes from the label "app" on the Workload.

Reply:
Also, not bad to link to Istio documentation for these labels terminologies.
https://istio.io/docs/setup/kubernetes/spec-requirements/